.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 extern
-__debug_bin
\ No newline at end of file
+__debug_bin
+*.test.output.*
```

Comment:
_shouldn't_ be necessary if we clean up after every test, but just wanted to ensure commit history remains clean without any accidental test files in it.

Reply:
ah i see why you had test.output extension now. there's no reason this needs to be here though if you use `t.TempDir()`